docs/wiring/blower.rst

```diff
@@ -0,0 +1,43 @@
+.. _Blower Harness:
+
+Blower Harness
+======================
+
+.. csv-table:: **BOM**
+   :widths: 5, 60, 5, 5, 5, 10, 10, 10
+   :header-rows: 1
+   :delim: 0009
+   :file: ./blower.bom.tsv
+
+.. image:: blower.png
+  :width: 800
+  :alt: Wiring diagram
+
+You will need all the tools listed.
+
+* Using items from above BOM as well as:
+   * 3mm heat shrink tubing [:ref:`ew-shrink3`]
+   * 5mm heat shrink tubing [:ref:`ew-shrink5`]
+   * Heat gun
+   * Molex Micro-Fit 3.0 Crimper
```

Comment:
Maybe we can have this tooling item in our database and link to it as well?

Reply:
Sure, I've added it to the database.  I also created a new category in the parts schema to differentiate tools (which should be bought once) from parts (which are bought per assembly).